test/unittests/execute_floating_point_test.hpp

```diff
@@ -13,7 +13,7 @@
 // Enables access and modification of the floating-point environment.
 // Here we use it to change rounding direction in tests.
 // Although required by the C standard, neither GCC nor Clang supports it.
-#pragma STDC FENV_ACCESS ON
+//#pragma STDC FENV_ACCESS ON
```

Comment:
Stray commit. This is a problem on mac, see #841.

Reply:
But does it go in? I think we can just keep "lessons learned" notes and remove all non-default rounding mode code from Fizzy.

Reply:
Well this change won't go into the PR as CI still works without it.

We can deal with this line in a separate PR. But all tests passing with even this turned off is weird.

Reply:
This is what C standard requires but compilers support is very limited. I.e. this is mostly ignored and e.g. GCC requires special flag `-frounding-math` to respect FP rounding mode.